azure-pipelines-ui-tests.dfe.yml

```diff
@@ -223,6 +223,55 @@ jobs:
       path: tests/robot-tests/test-results/
       artifactName: test-results-admin-public
 
+- job: 'PublicAPI'
+  displayName: Public API suite - Robot UI tests
+  timeoutInMinutes: 160
+  cancelTimeoutInMinutes: 10
+  condition: succeededOrFailed()
+  pool: ees-ubuntu2204-large
+  workspace:
+    clean: all
+  steps:
+  - checkout: self
+    clean: true
+    fetchDepth: 5
+    fetchTags: false
+
+  - task: UsePythonVersion@0
+    displayName: Use Python 3.10
+    timeoutInMinutes: 5
+    inputs:
+      versionSpec: 3.10
+
+  - task: AzureKeyVault@2
+    displayName: 'Azure Key Vault: s101d01-kv-ees-01'
+    inputs:
+      azureSubscription: $(SPN_NAME)
+      KeyVaultName: s101d01-kv-ees-01
+      SecretsFilter: ees-test-ADMIN-PASSWORD,ees-test-ANALYST-PASSWORD,ees-test-expiredinvite-password,ees-test-NOINVITE-PASSWORD,ees-test-PENDINGINVITE-PASSWORD,ees-alerts-slackapptoken
+      RunAsPreJob: true
+
+  - task: PythonScript@0
+    displayName: Admin public UI tests
```

Comment:
I think this needs to say "Admin Public API tests"?

Reply:
Thanks Dunc for quickly reviewing my PR, I think Public API UI tests may sound reasonable I guess - since the current UI tests have both admin and public user journeys.

Reply:
Oh yeah, good shout!  Approved :)